docs/articles/test-workflow-templates.md

```diff
@@ -1,34 +1,34 @@
 # Test Workflow Templates
 
-Test Workflow Templates allow you to define common properties/traits that you can reuse 
-consistently across your TestWorkflows. 
+Test Workflow Templates allow you to define building blocks which you can reuse across your workflows.
+Strategically, templates can bring standardization across your organization. You can define a tool chain which is easy to discover across teams.
+At a more tactical level, templates avoid repetition and can be used to hide implementation details.
 
-For example, with Test Workflow Templates you can easily 
-- define specific ENV variables to be passed to all containers.
-- define a common Git Source for your tests.
-- define common setup or tearDown steps for your tests.
-- define specific commands or images to use for your tests, while still allowing a Test Workflow to set
-  the version of the testing tool.
-- etc.
+Templates are flexible and can be used is various ways:
 
-## How do they work?
+- Template testing tools: Define a common usage of Cypress across your organization.
+- Template setup: Run scripts before your tests or tear down steps.
+- Template test sources: Define credentials once to make it easy to update them.
+- Template common labels: Attach labels per company department.
+- Template technical details: Define an abstraction which handles closing Istio sidecar.
+- Template integrations: Run anything after tests fail.
```

Comment:
some more suggestions:

- Template log collection: Collect logs from tested components to help with troubleshooting [read more](..)
- Template application-under-test: Start the actual components to be tested (and/or their dependencies) instead of relying those to be running